src/lib/swapper/swappers/ThorchainSwapper/utils/getUniqueAddressSubstring.test.ts

```diff
@@ -0,0 +1,53 @@
+import { ethChainId, toAssetId } from '@shapeshiftoss/caip'
+import { describe, expect, it } from 'vitest'
+
+import { getUniqueAddressSubstring } from './getUniqueAddressSubstring'
+
+const FINAL_ASSET_CONTRACT_ASSETID = toAssetId({
```

Comment:
Non-blocking: Same comment re: no need for `contract` vernacular anymore since this is an `AssetId`, implied not an EOA